src/components/attendee/SearchOverlay/EventItem/EventItem.tsx

```diff
@@ -0,0 +1,94 @@
+import { useCallback } from "react";
+import { useHistory } from "react-router-dom";
+import { Button } from "components/attendee/Button";
+import { NavOverlayTabType } from "components/attendee/NavOverlay/NavOverlay";
+import { differenceInCalendarDays } from "date-fns";
+
+import { SPACE_TAXON, STRING_DASH_SPACE, STRING_SPACE } from "settings";
+
+import { ScheduledEvent } from "types/venues";
+
+import { eventEndTime, eventStartTime } from "utils/event";
+import {
+  formatDateRelativeToNow,
+  formatTimeLocalised,
+  getDayMonthNow,
+} from "utils/time";
+import { enterSpace } from "utils/url";
+
+import { useSpaceById } from "hooks/spaces/useSpaceById";
+import { useWorldById } from "hooks/worlds/useWorldById";
+
+import CN from "../SearchOverlay.module.scss";
+
+export type EventItemProps = {
+  event: ScheduledEvent;
+  onClick: () => void;
+  setNavMenu: React.Dispatch<React.SetStateAction<string | undefined>>;
+};
+
+export const EventItem: React.FC<EventItemProps> = ({
+  event,
+  onClick,
+  setNavMenu,
+}) => {
+  const { space } = useSpaceById({ spaceId: event.spaceId });
+  const { world } = useWorldById({ worldId: event.worldId });
+
+  const { push: openUrlUsingRouter } = useHistory();
+
+  const handleEnterSpace = useCallback(() => {
+    enterSpace(world?.slug, space?.slug, {
+      customOpenRelativeUrl: openUrlUsingRouter,
+    });
+    onClick();
+  }, [world?.slug, openUrlUsingRouter, onClick, space?.slug]);
+
+  const showDate = Boolean(
+    differenceInCalendarDays(eventEndTime({ event }), eventStartTime({ event }))
+  );
+
+  const handleEnterSchedule = () => {
+    setNavMenu(NavOverlayTabType.schedule);
+  };
+
+  return (
+    <div>
+      <div className={CN.searchItemResultHeader}>
+        <h3 className={CN.searchItemResultTitle}>
+          {event.name}
+          <span>Experience</span>
+        </h3>
+      </div>
+      <div className={CN.searchItemResultSubtitle}>
+        <span>
+          {formatTimeLocalised(eventStartTime({ event })) + STRING_DASH_SPACE}
+        </span>
+        <span>
+          {showDate && formatDateRelativeToNow(eventEndTime({ event }))}
+        </span>
+        {STRING_SPACE}
+        <span>{formatTimeLocalised(eventEndTime({ event }))}</span>
+        <span> in {space?.name}</span>
+      </div>
+      <p className={CN.searchItemResultSubtitle}>{event.description}</p>
+      <div className={CN.actionContainer}>
+        <Button
+          variant="alternative"
+          border="alternative"
+          onClick={handleEnterSpace}
+          marginless
+        >
+          Go to {SPACE_TAXON.lower}
+        </Button>
+        <Button
+          variant="alternative"
+          border="alternative"
+          onClick={handleEnterSchedule}
+        >
+          See all experiences on {getDayMonthNow()}
```

Comment:
but experience is not necessarity happens today, is it correct that we always use current date even if the found experience is tomorrow, for example?

Reply:
So the behaviour for this isn't documented in the figma and I decided to show current date in the schedule as a solution

Reply:
But, after thinking about it, I've decided to change the date to be the event start day. This would effectively lead user to the schedule day of the event in question

Reply:
[09146a2](https://github.com/sparkletown/sparkle/pull/3074/commits/09146a2a8a1fd2ccbb351fd1056f0487e6181767)